cmd/system-probe/probe.go

```diff
@@ -45,9 +45,15 @@ func CreateSystemProbe(cfg *config.AgentConfig) (*SystemProbe, error) {
 	}
 
 	// make sure debugfs is mounted
-	mounted := util.IsDebugfsMounted()
-	if !mounted {
-		return nil, fmt.Errorf("%s: debugfs is not mounted and is needed for eBPF-based checks, run \"sudo mount -t debugfs none /sys/kernel/debug\" to mount debugfs", ErrTracerUnsupported)
+	err = util.IsDebugfsMounted()
+	if err != nil {
+		if os.IsPermission(err) {
+			return nil, fmt.Errorf("%s: system-probe does not have permission to access debugfs, please run system-probe with root permission", ErrTracerUnsupported)
```

Comment:
Just make this: `system-probe does not have permission to access debugfs`. 

We can remove the suggestion to run it as root (since technically all we need & want to recommend is configuring proper capabilities)

Reply:
I was about to say I'm not certain about the wording. Thanks for the suggestion.